src/main/java/com/minecolonies/entity/ai/basic/AbstractEntityAIInteract.java

```diff
@@ -79,9 +78,14 @@ protected final boolean mineBlock(@NotNull final BlockPos blockToMine, @NotNull
     {
         Block curBlock = world.getBlockState(blockToMine).getBlock();
         if (curBlock == null
-            || curBlock.equals(Blocks.air)
-            || BlockUtils.shouldNeverBeMessedWith(curBlock))
+              || curBlock.equals(Blocks.air)
+              || (curBlock.getMaterial().isLiquid())
+              || BlockUtils.shouldNeverBeMessedWith(curBlock))
         {
+            if(curBlock != null
+                 && curBlock.getMaterial().isLiquid()){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)